deployments/helm/nvidia-dra-driver-gpu/NOTICE

```diff
@@ -0,0 +1,4 @@
+Copyright 2025 NVIDIA CORPORATION
```

Comment:
cf. https://www.apache.org/legal/src-headers.html#notice

Reply:
Should this include the same / similar text as `# SPDX-FileCopyrightText:` in the license header?

Reply:
:) No.

The header facilitates discovering the license applied to a project/file in an automated fashion. It's typically used for SBOM generation / scanning / compliance purposes. 

The NOTICE file is an implementation detail of the AL2.